cpp/benchmarks/fixture/benchmark_fixture.hpp

```diff
@@ -51,14 +54,13 @@ namespace cudf {
  * BENCHMARK_REGISTER_F(my_benchmark, my_test_name)->Range(128, 512);
  */
 class benchmark : public ::benchmark::Fixture {
+  std::unique_ptr<rmm::mr::cnmem_memory_resource> mr{
```

Comment:
Why not go straight to using pool_memory_resource here, to accelerate dropping cnmem?

Reply:
Yeah, I had the same thought. Wanted to make sure people were okay with it before I made such a change.

Reply:
Lets do this in a follow up PR so we can have further discussion if desired.